src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -125367,16 +125522,13 @@ void M(object? o1)
         // 2 states * 3 states * 3 states = 18 combinations
         [InlineData("locationNonNull", "valueNonNull", "comparandNonNull", true)]
         [InlineData("locationNonNull", "valueNonNull", "comparandMaybeNull", true)]
-        [InlineData("locationNonNull", "valueNonNull", "null", true)]
```

Comment:
Why was this case removed? This reads to me as: "if the location contains null, write non-null to it."

Reply:
Those three scenario were moved to their own test below (for now). Currently `null` literals do not contribute their nullability to method type re-inference, so the diagnostics are different (ie. worse).

---
In reply to: [493903400](https://github.com/dotnet/roslyn/pull/47952#discussion_r493903400) [](ancestors = 493903400)

Reply:
Sorry, I must not have noticed it below.